### PR TITLE
Added NFToken documentation

### PR DESCRIPTION
**Pull request by mialbu**
Added NFToken documentation and changed old addresses to addresses with version 0x35.

### CODE REVIEW

File: docs/neo3_guides/token_transfer.md [hunk 1]
```diff
@@ -91,9 +91,9 @@ signers and the wallet to the `TransactionBuilder`, so it is not necessary to co
 manually.
 
 ```java
-Account account1 = Account.fromWif("L1WMhxazScMhUrdv34JqQb1HFSQmWeN2Kpc1R9JGKwL7CDNP21uR");
-Account account2 = Account.fromWif("L45BGYyybk91pvwH3Mj1CfDZ11GGQLVPr6qfzpWugeP4WeJZyfki");
-Account account3 = Account.fromWif("L2pN4EbagTuk9Kiib8sjRmMQznxqCVEs1HR8DRaxmnPicjg9FdNc");
+Account account1 = Account.fromWIF("L1WMhxazScMhUrdv34JqQb1HFSQmWeN2Kpc1R9JGKwL7CDNP21uR");
```
Comment (gsmachado) [inline docs/neo3_guides/token_transfer.md:94]: These WIF are still compatible with neow3j? After we changed the default `AddressVersion`?

Comment (mialbu) [reply docs/neo3_guides/token_transfer.md]: They actually are, but while you commented, I was just updating some addresses in the docs and adjusted accounts that are created using the method `fromWIF()` to the ones used in `neo3-privatenet-docker`.

**Review by gsmachado — APPROVED** [review]

File: docs/neo3_guides/token_transfer.md [hunk 2]
```diff
@@ -146,4 +145,28 @@ in [this Section](neo3_guides/contract_invocation.md#signing-a-transaction-with-
 
 ## Non-fungible Tokens (NEP-11)
 
-?> _In the works_
+neow3j provides a wrapper class `NFToken` that can interact with token contracts that support the [NEP-11](https://github.com/neo-project/proposals/pull/41) standard.
+
+> The NEP-11 standard is not yet final, so there might still be changes to it and there does not exist an implementation for it yet.
```
Comment (csmuller) [inline docs/neo3_guides/token_transfer.md:150]:
```suggestion
> The NEP-11 standard is not yet final, so there might still be changes to it.

Comment (csmuller) [reply docs/neo3_guides/token_transfer.md:150]: I thought we saw some examples somewhere, no?

File: docs/neo3_guides/token_transfer.md [hunk 3]
```diff
@@ -146,4 +145,28 @@ in [this Section](neo3_guides/contract_invocation.md#signing-a-transaction-with-
...
+
+As shown above in the structure, the `NFToken` is a subtype of the class `Token`, so it utilizes of the methods `getName()`, `getSymbol()`, `getTotalSupply()` and `getDecimals()`.
```
Comment (csmuller) [inline docs/neo3_guides/token_transfer.md:152]:
```suggestion
As shown above in the structure, the `NFToken` is a subtype of the class `Token`, so it utilizes the methods `getName()`, `getSymbol()`, `getTotalSupply()` and `getDecimals()`.
```

**Review by csmuller — CHANGES_REQUESTED** [review]

File: docs/neo3_guides/token_transfer.md (hunk 2 above)
Comment (mialbu) [reply docs/neo3_guides/token_transfer.md:150]: True, I'll remove it.

**Review by csmuller — APPROVED** [review]

**Comment by gsmachado** [comment]
@mialbu @csmuller shouldn't this go to `master` branch?

**Comment by csmuller** [comment]
> @mialbu @csmuller shouldn't this go to `master` branch?

Yes you're right. I assumed that the NFT implementation was in the current milestone but it was part of the previous one.

**Pull request merged**